Tests/Legacy/Shopware/Connect/ConnectExportTest.php

```diff
@@ -126,6 +129,8 @@ public function testExport()
      */
     public function testExportErrors()
     {
+        Shopware()->Db()->executeUpdate('UPDATE s_filter SET sortmode = 1');
```

Comment:
is it possible to move this ConnectExportTest in new integration structure or there is something which is not compatible?

Reply:
tried it but then the tests will fail I don't know why